components/AppBar/src/MDCAppBarNavigationController.h

```diff
@@ -131,6 +131,14 @@ __attribute__((objc_subclassing_restricted)) @interface MDCAppBarNavigationContr
 - (nullable MDCAppBarViewController *)appBarViewControllerForViewController:
     (nonnull UIViewController *)viewController;
 
+/**
+ A block that is assigned to each assigned to each @c MDCAppBarViewController's
+ @c traitCollectionDidChange:.
+ */
+@property(nonatomic, copy, nullable) void (^defaultTraitCollectionDidChangeBlock)
```

Comment:
is there a reason why did has the default prefix?

Reply:
They can be overridden on the `MDCAppBarViewController` itself.  I felt that the "default" conveyed that a bit better.  I'm not sold on the name and would change it if you wanted me to.

Reply:
oh, are you saying that if we call it just traitCollectionDidChangeBlock it would override the original block?

Reply:
Sorry, no. I mean that once the `MDCAppBarViewController` is created, the VC it is a child of could potentially override the block.  I was trying to convey the idea that this property isn't affecting this object, but is going to be assigned to the `traitCollectionDidChangeBlock` of the `MDCAppBarViewController` objects that it generates.

I'm pretty sure I've done it poorly (judging by this conversation), so I'm open to suggestions. I know `traitCollectionDidChangeBlock` is a name meant for the object itself (when it receives its own `traitCollectionDidChange:` message).

Reply:
ahh got it! What if we say traitCollectionDidChangeBlockForAppBar: or is that too long winded?

Reply:
There's no such thing as "long-winded" in Objective-C! ;-)

Reply:
Done.